server/sandbox.go

```diff
@@ -360,11 +368,12 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 		return nil, err
 	}
 
-	podInfraContainerName := sb.name + "-infra"
-	var podInfraContainer *oci.Container
+	podInfraContainer := sb.infraContainer
+	containers := sb.containers.List()
+	containers = append(containers, podInfraContainer)
```

Comment:
Hi, We need to stop/delete infra container, right? I cannot find other place that does this work in RemovePodSandbox.